src/axom/spin/BVH.hpp

```diff
@@ -168,14 +177,11 @@ class BVH
    * \pre boxes != nullptr
    * \pre numItems > 0
    */
-  BVH(const FloatType* boxes,
-      IndexType numItems,
-      int allocatorID = axom::execution_space<ExecSpace>::allocatorID());
+  int initialize(const BoxType* boxes,
```

Comment:
Looks good.  I'm OK with just adding the arguments to a ctor, but the disadvantage is that the ctor can throw exceptions with that approach.